### DISCUSSION TITLE
List created but the windows freezes!

### DISCUSSION THREAD

**Discussion by iRupam** [community/community#76291] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I tried to add a repository to a list, but after clicking Create, the modal seems to be feezed. I checked and found out that the list is created and the repo is added successfully, but the moal is still in freeze status.

Found out this issue is reported a few months ago [here](https://github.com/orgs/community/discussions/58574).

![github list freeze](https://github.com/community/community/assets/27852445/f9b21593-b358-44c7-b1ff-ebefa63fb255)

**Comment by Jamim** [comment]
This discussion duplicates #50880, just as a dozen of other reports.
